packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -304,17 +312,27 @@ export class TransactionHandler {
     const startTime = Date.now()
     if (retry) {
       ;(async () => {
+        const connections =
```

Comment:
let's include the original connection when retrying too

Reply:
yeah i push it to this list on line 319. wasnt sure if it was necessarily included in fallbackConnections so i filtered it out and re-added it. also ensures the list is at least len 1

Reply:
ooh i see yea it is you can just use fallbackConnections
https://github.com/AudiusProject/audius-protocol/blob/main/packages/libs/src/services/solana/SolanaWeb3Manager.ts#L190-L207

Reply:
oo nice adding that in now